tests/xgboost/test_xgboost_model_export.py

```diff
@@ -49,6 +49,16 @@ def xgb_model():
     return ModelWithData(model=model, inference_dataframe=X, inference_dmatrix=dtrain)
 
 
+@pytest.fixture(scope="session")
+def xgb_sklearn_model():
+    boston = datasets.load_boston()
```

Comment:
Can we use a different dataset because `load_boston` is deprecated?

https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html:

> DEPRECATED: load_boston is deprecated in 1.0 and will be removed in 1.2.

Reply:
Thanks for your suggestion! I didn't notice that boston will be removed. I use the wine dataset in the latest commit.